spec2sdk/models/converters.py

```diff
@@ -37,24 +35,10 @@
 converters = Registry()
 
 
-class CommonFields(TypedDict):
-    name: str | None
-    description: str | None
-    default_value: Any
-
-
-def convert_common_fields(data_type: DataType) -> CommonFields:
-    return CommonFields(
-        name=make_class_name(data_type.name) if data_type.name else None,
-        description=data_type.description,
-        default_value=data_type.default_value,
-    )
-
-
 @converters.register(predicate=is_instance(StringDataType))
 def convert_string(data_type: StringDataType) -> StringType:
     return StringType(
-        **convert_common_fields(data_type),
+        name=make_class_name(data_type.name) if data_type.name else None,
```

Comment:
Maybe one could put this into a small helper function like `get_name(data_type) -> Optional[str]` or something?

Reply:
Done.